var/spack/repos/builtin/packages/r-utils/package.py

```diff
@@ -0,0 +1,38 @@
+##############################################################################
+# Copyright (c) 2013-2017, Lawrence Livermore National Security, LLC.
+# Produced at the Lawrence Livermore National Laboratory.
+#
+# This file is part of Spack.
+# Created by Todd Gamblin, tgamblin@llnl.gov, All rights reserved.
+# LLNL-CODE-647188
+#
+# For details, see https://github.com/llnl/spack
+# Please also see the NOTICE and LICENSE files for our notice and the LGPL.
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU Lesser General Public License (as
+# published by the Free Software Foundation) version 2.1, February 1999.
+#
+# This program is distributed in the hope that it will be useful, but
+# WITHOUT ANY WARRANTY; without even the IMPLIED WARRANTY OF
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the terms and
+# conditions of the GNU Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public
+# License along with this program; if not, write to the Free Software
+# Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA 02111-1307 USA
+##############################################################################
+from spack import *
+
+
+class RUtils(RPackage):
+    """Utility functions useful when programming and
+    developing R packages."""
+
+    homepage = "https://github.com/HenrikBengtsson/R.utils"
+    url      = "https://cran.rstudio.com/src/contrib/R.utils_2.5.0.tar.gz"
+    list_url = "https://cran.r-project.org/src/contrib/Archive/R.utils"
+
+    version('2.5.0', 'a728ef3ceb35cafc4c39ea577cecc38b')
+
+    depends_on('r-oo', type=('build', 'run'))
```

Comment:
Can you please either point out dependencies in the description, or gather related PRs together? This is just to avoid for instance merging this - as it looks completely fine - and leaving #5962 (which is a dependency) out for any reason.